File: OCSwiftDemo/OCSwiftDemo/ViewController.m
//
//  ViewController.m
//  OCSwiftDemo
//
//  Created by plz on 2021/1/27.
//

#import "ViewController.h"
#import "OCSwiftDemo-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    SwiftTest *swift =[[SwiftTest alloc]init];
    [swift test];
    [swift sayHello];
    
    UILabel * titleLabel=[[UILabel alloc]initWithFrame:self.view.bounds];
    titleLabel.textColor=UIColor.whiteColor;
    titleLabel.backgroundColor = UIColor.redColor;
    titleLabel.textAlignment = NSTextAlignmentCenter;
    [self.view addSubview:titleLabel];
    titleLabel.text = [swift getNameAndAgeWithName:@"plz" age:26];

}


@end

File: SwiftOCDemo/SwiftOCDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftOCDemo
//
//  Created by plz on 2021/1/27.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let teacher = LZTeacher()
        print(teacher)
        
        let label = UILabel.init(frame: self.view.bounds)
        label.backgroundColor = UIColor.red
        self.view.addSubview(label)
        
        let ocTest = OCTest()
        ocTest.sayHello()
        
        label.textColor = UIColor.white
        label.textAlignment = NSTextAlignment.center
        label.text = ocTest.title()
    }
}

class LZTeacher {
    var name = "plz"
    var age = 20
}
